pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -268,6 +274,17 @@ func getSubvolumeGroupName(cephFilesystemSubVolumeGroup *cephv1.CephFilesystemSu
 	return cephFilesystemSubVolumeGroup.Name
 }
 
+func (r *ReconcileCephFilesystemSubVolumeGroup) pinSubvolumeGroup(cephFilesystemSubVolumeGroup *cephv1.CephFilesystemSubVolumeGroup) error {
```

Comment:
This method doesn't add much, I would suggest get rid of this method and just call `cephclient.PinCephFSSubVolumeGroup()` above on line 259.